bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java

```diff
@@ -200,6 +203,8 @@ protected void initModel() {
                         }
                         dmImageUrl = dmServerUrl + dmAssetName;
                     }
+                    useWOID = useWOID && StringUtils.isEmpty(policyDelegatePath);
```

Comment:
Not sure about the intention here

Reply:
@vladbailescu  as per my understanding here, users as the option to provide policy in the GET request as well, as mentioned here [0]. So IMO if thats the case, we should not use WOID and fallback to AIS ?


[0] https://github.com/adobe/aem-core-wcm-components/blob/main/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java#L227